wasm/src/wasm_builtins.rs

```diff
@@ -38,3 +42,14 @@ pub fn builtin_print(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
     Ok(vm.get_none())
 }
+
+pub fn builtin_log(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
```

Comment:
Stick to the name buildin_print, since this is the builtin function name.

Reply:
Alright, I renamed them to `builitin_print_html` and `builtin_print_console` to differentiate between the two.